pipelines/TechAgnostic/Utility/ConvertToHailMT.wdl

```diff
@@ -8,7 +8,7 @@ import "../../../tasks/VariantCalling/GLNexus.wdl" as GLNexus
 import "../../../tasks/Utility/Hail.wdl" as Hail
 import "../../../tasks/Utility/Finalize.wdl" as FF
 
-workflow ConvertToHailMT {
+workflow RunConvertToHailMT {
```

Comment:
Why is this necessary? It might break things.

Reply:
Miniwdl complained that the task being called shouldn't be the same as the workflow name. This wdl is calling call Hail.**ConvertToHailMT** and its workflow name is **ConvertToHailMT**. I could try renaming the task being called `as` something else

Reply:
yep, miniwdl is fine if the call name uses an alias. I'll make that change instead to avoid breaking changes

Reply:
Thanks. I'm just afraid that someone is using the workflow and it suddenly disappear for them.